CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-*
+* Add CSV mixin to wrap `ripe-commons-js` functions - related to [#302](https://github.com/ripe-tech/ripe-pulse/issues/302)
```

Comment:
ripe-pulse/#302

Reply:
Welp, that's clearly wrong :) 

Nice catch, thanks! It's another #302 clearly :D